src/components/Quote.test.js

```diff
@@ -0,0 +1,9 @@
+import renderer from 'react-test-renderer';
+import Quote from './Quote';
+
+describe('Test Quotes component', () => {
+  it('Test component should render properly', () => {
+    const tree = renderer.create(<Quote />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
```

Comment:
- [ ] Some tests are failing, let me know if there is anything I didn't do.

![test](https://user-images.githubusercontent.com/53124548/213652761-3df287e1-5686-4ccc-bc70-db9d09abed1f.png)

Reply:
Fixed it thank you @elisha2kyakpo1